src/routing/http-routing-v1.md

```diff
@@ -97,7 +100,23 @@ This API does not support pagination, but optional pagination can be added in a
 
 ## Streaming
 
-This API does not currently support streaming, however it can be added in the future through a backwards-compatible update by using a content type other than `application/json`.
+This API supports streaming requests made by sending an `Accept` HTTP Header containing
+`application/x-ndjson`. The response will be formatted as [Newline Delimited JSON (ndjson)](http://ndjson.org/),
+with one *read provider record* per line:
+
+
+```json
+{"Protocol": "<protocol_name>", "Schema": "<schema>", ...}
+{"Protocol": "<protocol_name>", "Schema": "<schema>", ...}
+{"Protocol": "<protocol_name>", "Schema": "<schema>", ...}
+...
+```
```

Comment:
We may return result with schema without `Protocol`, let's simplify

````suggestion
```json
{"Schema": "<schema>", ...}
{"Schema": "<schema>", ...}
{"Schema": "<schema>", ...}
...
```
````

Reply:
Okay, but then we have to update the rest of the spec. That is a bigger change than just adding streaming.

Reply:
https://github.com/ipfs/kubo/pull/9877/files#r1203865317

Reply:
As per https://github.com/ipfs/kubo/pull/9877/files#r1205372011, we will address this in a separate IPIP.